### DISCUSSION TITLE
失效网站模板提交区

### DISCUSSION THREAD

**Discussion by a76yyyy** [qd-today/templates#272] [General]
有网站关站或模板已彻底失效的，请提交在这里，待确认后从仓库中删除。

格式如下

```Markdown
- 名称：模板 HAR 或者网站名称
- 网址：模板 HAR 或者网站地址
- 模板评论区：模板评论区地址
- 描述：失效模板或网站的相关描述
```

**Comment by OwnerCM** [comment]
名称：CSDN-专业IT技术论坛.har
网址：www.csdn.com
描述：csdn已下线签到活动

**Reply by a76yyyy** [reply]
已删除

**Comment by kokpk** [comment]
名称：漫画补档(账号密码版)
网址：https://www.manhuabudang.com
描述：网站已打不开，新网址为https://www.manhuabudangbbs.com/

**Reply by a76yyyy** [reply]
已更新

**Reply by kokpk** [reply]
我更新了试了下提示
Failed at 1/4 request,
Fail assert: {"re": "您没有登录或者您没有权限访问此页面", "from": "content"} from failed_asserts,
Request URL: https://www.manhuabudangbbs.com/u.php
网站现在改名叫补档冰室界面跟原来的看着差不多但账号登录有验证码 我手动打开这个链接https://www.manhuabudangbbs.com/u.php是正常的。

**Reply by a76yyyy** [reply]
@wjf0214 

> 我更新了试了下提示 Failed at 1/4 request, Fail assert: {"re": "您没有登录或者您没有权限访问此页面", "from": "content"} from failed_asserts, Request URL: https://www.manhuabudangbbs.com/u.php 网站现在改名叫补档冰室界面跟原来的看着差不多但账号登录有验证码 我手动打开这个链接[https://www.manhuabudangbbs.com/u.php是正常的。](https://www.manhuabudangbbs.com/u.php%E6%98%AF%E6%AD%A3%E5%B8%B8%E7%9A%84%E3%80%82)

**Reply by wjf0214** [reply]
@kokpk 我这个是cookie版本的，cookie有没有抓取对。

**Reply by kokpk** [reply]
我前面是用插件自动抓的，我手动抓了输入进去提示一样

**Comment by kokpk** [comment]
名称：不移之火
网址：www.byzhihuo.com
描述：公共模版中2023-01-01 10:00:00更新的这个显示签到成功实际未成功，评论区里的可用。

**Reply by a76yyyy** [reply]
已更换

**Comment by OwnerCM** [comment]
名称：吾爱汇编
网站：https://www.52hb.com/
描述：网站已取消签到任务

**Reply by a76yyyy** [reply]
暂不删除

**Comment by OwnerCM** [comment]
名称：大众点评
网站：https://m.dianping.com/
描述：已取消签到活动

**Reply by a76yyyy** [reply]
已删除

**Comment by OwnerCM** [comment]
名称：看雪安全论坛
网站：https://bbs.pediy.com/
描述：论坛更换域名

**Reply by a76yyyy** [reply]
已更新

**Comment by 201SAndvic** [comment]
名称：微妙网
网址：https://www.wmiao.com/
描述：日志：Failed at 1/4 request,
Fail assert: {"re": "注册", "from": "content"} from failed_asserts,
Request URL: https://www.wmiao.com/

**Reply by wjf0214** [reply]
已更新，[微妙网.har](https://raw.githubusercontent.com/wjf0214/qd-templates/master/微妙网.har)

**Comment by vipwuping** [comment]
名称：宽带技术网.har
网址：www.chinadsl.net
描述：旧版本用的是HTTP 模板没有更新 新版网站已开启 https

**Reply by wjf0214** [reply]
上月更过，[宽带技术网.har](https://raw.githubusercontent.com/wjf0214/qd-templates/master/宽带技术网.har)

**Comment by coffeefirst** [comment]
名称：迅维
- 网址：https://www.chinafix.com/
- 描述：似乎用户名密码登录有问题，因为OCR？能否改成cookie登录

**Reply by a76yyyy** [reply]
可以直接用 @wjf0214 这位大佬[仓库](https://github.com/wjf0214/qd-templates)里的cookie版么

**Reply by coffeefirst** [reply]
> 可以直接用 @wjf0214 这位大佬[仓库](https://github.com/wjf0214/qd-templates)里的cookie版么

可用，谢谢

**Reply by coffeefirst** [reply]
掌上百科pdawiki.com账号登录也有问题，不知道有没有cookie版？

**Comment by coffeefirst** [comment]
名称：掌上百科

网址：https://www.pdawiki.com/
描述：签到报错，能否改成cookie登录？

**Comment by sdlkhfksl** [comment]
名称：189天翼云账号密码版
- 网址：https://cloud.189.cn
- 描述：各种报错，Failed at 1/20 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received,
Request URL: https://cloud.189.cn/api/portal/loginUrl.action?redirectURL=https%3A%2F%2Fcloud.189.cn%2Fweb%2Fredirect.html%3FreturnURL%3D%2Fmain.action

Failed at 15/20 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to api.cloud.189.cn:443 ,
Request URL: https://api.cloud.189.cn/mkt/userSign.action?rand={{_timestamp1|urlencode}}&clientType=TELEANDROID&version=8.6.3&model=SM-G930K

Failed at 16/20 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: https://m.cloud.189.cn/v2/drawPrizeMarketDetails.action?taskId=TASK_SIGNIN&activityId=ACT_SIGNIN

Failed at 2/20 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: {{saomaurl}}

**Reply by a76yyyy** [reply]
这个是请求超时了，一般是网络问题或者服务器未响应

**Comment by 8MiYile** [comment]
- 名称：好家伙机场
- 网址：https://haojiahuo.live/
- 模板评论区：https://github.com/qd-today/templates/issues/92
- 描述：在准备添加的时候，想要先注册用户，发现网站不存在了，如果网站已经不做了，就移除吧

**Reply by a76yyyy** [reply]
已删除

**Comment by SupercoolBD** [comment]
- 名称：m-teams.cc
- 网址：https://github.com/qd-today/templates/pull/291
- 描述：報錯
```
2024-01-20 22:04:52
日志：Failed at 1/2 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Failed to connect to flaresolverr port 8191 after 3058 ms: Couldn't connect to server,
Request URL: {{flaresolverr_url}}
```
'15cm/flaresolverr' container的日志
```
024-01-20 14:04:18 INFO     ReqId 140454670890816 FlareSolverr 3.1.2

2024-01-20 14:04:18 DEBUG    ReqId 140454670890816 Debug log enabled

2024-01-20 14:04:18 INFO     ReqId 140454670890816 Testing web browser installation...

2024-01-20 14:04:18 INFO     ReqId 140454670890816 Platform: Linux-5.15.0-91-generic-x86_64-with-glibc2.31

2024-01-20 14:04:18 INFO     ReqId 140454670890816 Chrome / Chromium path: /usr/bin/chromium

find: ‘/root/.config/chromium/Crash Reports/pending/’: No such file or directory

2024-01-20 14:04:22 INFO     ReqId 140454670890816 Chrome / Chromium major version: 113

2024-01-20 14:04:22 INFO     ReqId 140454670890816 Launching web browser...

2024-01-20 14:04:22 DEBUG    ReqId 140454670890816 Launching web browser...
```

**Reply by a76yyyy** [reply]
非模板失效问题，599是因为容器间网络连通性问题导致

**Comment by 125540947** [comment]
吾爱：
提示Failed at 8/9 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://www.52pojie.cn/home.php?mod=spacecp&ac=credit&showcredit=1&inajax=1&ajaxtarget=extcreditmenu_menu

**Reply by RUTHDOMINGUEZ8216** [reply]
對。我試也失效了

**Comment by a76yyyy** [comment]
> 吾爱：
> 提示Failed at 8/9 request,
> Fail assert: {"re": "200", "from": "status"} from success_asserts,
> Response Error : HTTP 302: Found,
> Request URL: https://www.52pojie.cn/home.php?mod=spacecp&ac=credit&showcredit=1&inajax=1&ajaxtarget=extcreditmenu_menu

请按照失效模板提交格式编辑内容

**Comment by nathandingx** [comment]
名称：吾爱破解
网址：www.52pojie.cn
描述：无法签到，具体原因不明

测试页报错如下:

Failed at 7/9 request,
Fail assert: {"re": "领取今日签到奖励", "from": "content"} from failed_asserts,
Request URL: https://www.52pojie.cn/forum.php

**Comment by 125540947** [comment]
- 名称：4550.fun 
- 网址：4550.fun 
- 模板评论区：模板评论区地址
- 描述：Failed at 1/6 request,
Fail assert: {"re": "欢迎您", "from": "content"} from success_asserts,
Response Error : HTTP 599: Recv failure: Connection reset by peer,
Request URL: https://4550.fun/member.php?mod=logging&action=login&loginsubmit=yes&inajax=1

**Comment by klcb2010** [comment]
名称：远景论坛
网址：https://i.pcbeta.com/home.php?mod=task
描述：无法签到，
具体原因不明 测试页报错如下:  Failed at 2/7 request,
Fail assert: {"re": "欢迎", "from": "content"} from success_asserts,
Request URL: https://bbs.pcbeta.com/member.php?mod=logging&action=login&loginsubmit=yes&inajax=1

**Comment by tophtab** [comment]
- 名称：DJI大疆社区
- 网址：https://bbs.dji.com/
- 模板评论区：https://github.com/qd-today/templates/issues/246
- 描述：老版本失效，我已在评论区提交新HAR，但因权限，无法编辑到主楼

**Comment by gmcnmm** [comment]
名称：bbs9091/bbs9090.com
网址：(http://bbs9090.com/sign.html)
模板评论区：https://github.com/qd-today/templates/issues/1076
描述：日志虽然显示为成功，但实际未签到
